### PR TITLE
fix benchmarks for jenkins

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

**Comment by chaudum** [comment]
@kovrus and @Autophagy can you take a look please

File: api/config.toml [hunk 1]
```diff
@@ -1,6 +1,8 @@
-debug = true
+debug = false
 crate_hosts = [
-  "localhost:4200",
+  "crate1:4200",
+  "crate2:4200",
+  "crate3:4200",
 ]
 
 [groups]
```
Comment (kovrus) [inline api/config.toml]: do we need to remove temporarily joins stuff from here too?

Comment (chaudum) [inline api/config.toml]: yes, we should disable JOINS graphs for now too

Comment (chaudum) [inline api/config.toml]: @kovrus this should be done in crate-web tho

**Pull request merged**